go.mod

```diff
@@ -469,3 +469,5 @@ exclude github.com/mattn/go-sqlite3 v2.0.3+incompatible
 
 // Use our fork xorm. go.work currently overrides this and points to the local ./pkg/util/xorm directory.
 replace xorm.io/xorm => github.com/grafana/grafana/pkg/util/xorm v0.0.1
+
+replace github.com/grafana/grafana-plugin-sdk-go => github.com/grafana/grafana-plugin-sdk-go v0.227.1-0.20240426134450-5fe9f7b9dfd4
```

Comment:
Good question. When new release of SDK is tagged the replace is removed and go.mod files updated in each module. I guess you mean for the purpose of testing these changes?

Reply:
It would be nice with a similar command `go mod edit -replace github.com/grafana/grafana-plugin-sdk-go=github.com/grafana/grafana-plugin-sdk-go@5fe9f7b9dfd42396cc596c72084afbb3ffc39f28` for `go work`. Guess that doesn't exist?

Reply:
I did update go.work instead. Was a bit weird since I first had to add it to go.mod with a go mod edit as above and then `go mod tidy` and then move the replace line into go.work.